parsers/fortran.c

```diff
@@ -582,10 +582,11 @@ static void makeLabelTag (vString *const label)
 
 static lineType getLineType (void)
 {
-	vString *label = vStringNew ();
+	static vString *label = NULL;
```

Comment:
> How do you want to achieve this using stack allocation?

It was just an example, but at a glance just using an automatic character array variable appears like it would work, since the code seems to limit the string to around 6-8 characters or something. Another example would be just recycling small allocations like (AFAIK) glibc's allocator already does.

Anyway, I only commented because a) it is an actual memory leak (albeit intentional), and b) it's a code-smell (IMO). It's certainly not a big deal, especially if it's a widely used pattern already.